classes/class-twenty-twenty-one-custom-colors.php

```diff
@@ -170,8 +170,9 @@ public function body_class( $classes ) {
 			$classes[] = 'is-background-light';
 		}
 
+		$should_respect_color_scheme  = get_theme_mod( 'respect_user_color_preference', true );
```

Comment:
Looks like this is flagging the phpstan check, but we do the same thing with `get_theme_mod( 'display_title_and_tagline', true )` — is this safe to `@phpstan-ignore-line`?

Reply:
Yes, it's safe to ignore :+1: